core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -419,6 +419,11 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
   final def get[E1 >: E, B](implicit ev1: E1 =:= Nothing, ev2: A <:< Option[B]): ZIO[R, Unit, B] =
     ZIO.absolve(self.mapError(ev1).map(ev2(_).toRight(())))
 
+  /**
+   * Returns a new effect that ignores the success or failure of this effect.
+   */
+  final def ignore: ZIO[R, Nothing, Unit] = self.either.unit
```

Comment:
Because this is the core `ZIO` type, I'd make a high-performance implementation:

```scala
self.foldCauseM(_ => ZIO.unit, _ => ZIO.unit)
```